### DISCUSSION TITLE
怎么才能拐回家呢？在线等，急！！！

### DISCUSSION THREAD

**Discussion by BITE-7274** [share121/inter-knot#1025] [其他]
![20240714174158](https://github.com/user-attachments/assets/19cb1cce-b1ba-45ef-8c2b-49ffb169fdb5)

**Comment by Qiling-XwX** [comment]
打晕，装进麻袋，直接跑路

**Comment by Ryan83602** [comment]
上面的,6

**Comment by BITE-7274** [comment]
@Qiling-XwX 已带回，这个怎么办·-·感觉我会被装进麻袋
![20240719214434](https://github.com/user-attachments/assets/a146a360-73a2-4d74-ad6e-9b2665b31960)

**Reply by Qiling-XwX** [reply]
?

**Reply by Qiling-XwX** [reply]
![58995C5D281B5E33C6F9A24F1F80D813](https://github.com/user-attachments/assets/1ac68058-7e1d-4ec2-b9ea-cabc28070308)
你等着，我马上去找你

**Reply by BITE-7274** [reply]
(*Φ皿Φ*)吓到我了，可恶，维多利亚家政是吧，我要让法厄同用大批账户给你刷差评：（

**Reply by Qiling-XwX** [reply]
维多利亚家政马上到你家门口！请放弃无谓的抵抗！
![6E372E1E04A1AA98A15B91F45F06436B](https://github.com/user-attachments/assets/d18fb302-3780-4369-ad5e-6051cfe62d66)

**Reply by BITE-7274** [reply]
可恶啊！你别过来！我...我可是有牲鬼在这！！！小心我和你同归于尽ε=( o｀ω′)ノ

**Reply by Qiling-XwX** [reply]
就此投降，没有人会受伤！
![38B2843718D7F614ACB3B77A54871523](https://github.com/user-attachments/assets/9a142519-baba-4e03-b6a6-6fc3a9a2f59a)

**Comment by ZhuYuan-148** [comment]
咳咳，这位市民还请遵纪守法

**Comment by Joker-JYH** [comment]
我已经打电话给治安局了，楼主自首吧

**Comment by BITE-7274** [comment]
@ZhuYuan-148 遭了！是治安官，不管了，还有一个麻袋，装走！！！

**Comment by Joker-JYH** [comment]
坏了，治安局问我怎么知道的，完蛋兄弟们，先跑路了

**Comment by BITE-7274** [comment]
别跑 https://github.com/Joker-JYH 传奇绳匠法厄同已经告诉我你的位置了！

**Comment by APEX-114514** [comment]
你是懂拍摄角度的

**Comment by Rope-Worker** [comment]
我已经打电话给治安局了，楼主自首吧

**Comment by BITE-7274** [comment]
楼上的 我已经知晓你的位置https://github.com/Rope-Worker 不想被麻袋装走就给我乖乖的去跟治安局说你报假警

**Comment by BITE-7274** [comment]
@Qiling-XwX 别过来！！！再过来我可就：）
![20240721175453](https://github.com/user-attachments/assets/bb177771-0987-4534-bda7-63991802a6bd)

**Comment by GanlinSama** [comment]
天哪，近距离目击犯罪现场！
偷偷拍个照记个脸，等以后治安局发通缉令了说不定能小赚一笔，嘻嘻(●'◡'●)
